src/components/display/fewlines/SnackBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+import { IdentityTypes } from "../../../@types/Identity";
+
+export const SnackBar: React.FC<{ type: IdentityTypes }> = ({ type }) => {
+  let method;
+  type.toUpperCase() === IdentityTypes.EMAIL
+    ? (method = "mail")
+    : (method = "SMS");
+  return <Snack>Confirmation {method} has been sent</Snack>;
```

Comment:
Hmmm. I'll let fenn discuss this.

Reply:
```suggestion
  const method = type.toUpperCase() === IdentityTypes.EMAIL ? "mail" : "SMS";
  return <Snack>Confirmation {method} has been sent</Snack>;
```
at least.

But I think we could create two versions:

```
  if (type.toUpperCase() === IdentityTypes.EMAIL) {
    return <Snack>Confirmation email has been sent</Snack>
  }
  if (type.toUpperCase() === IdentityTypes.PHONE) {
    return <Snack>Confirmation SMS has been sent</Snack>
  }
  return null
```

This might be more verbose but it's really easier to edit in the future and to replace by a translated string for each

Reply:
OK fixed! cd6f1c4